subprojects/gst-plugins-bad/sys/msdk/gstmsdkh264enc.c

```diff
@@ -860,12 +870,23 @@ gst_msdkh264enc_class_init (GstMsdkH264EncClass * klass)
       "Intel MSDK H264 encoder", "Codec/Encoder/Video/Hardware",
       "H264 video encoder based on " MFX_API_SDK,
       "Josep Torra <jtorra@oblong.com>");
-  gst_element_class_add_static_pad_template (element_class, &src_factory);
+
+  gst_element_class_add_pad_template (element_class,
+      gst_pad_template_new ("sink", GST_PAD_SINK, GST_PAD_ALWAYS,
+          cdata->sink_caps));
+  gst_element_class_add_pad_template (element_class,
+      gst_pad_template_new ("src", GST_PAD_SRC, GST_PAD_ALWAYS,
+          cdata->src_caps));
```

Comment:
It's already OK, but it's better to add some "gst_pad_template_set_documentation_caps" like we do for VA plugins.

Reply:
Thank you, @HeJunyan, I will try to add the function later.

Reply:
Add new patch: [msdkenc: Add documented capabilities to expose "stable" caps](https://github.com/intel-media-ci/gstreamer/pull/51/commits/221afa70fdcfa7090d6d613d6e7c470f2f34ddf2)